File: food_order_app/lib/data/requests/update_order_with_shipping_info_request.dart
class UpdateOrderWithShippingInfoRequest {
  int orderId;
  String shippingAddress;
  double shippingFee;
  String shippingPhoneNumber;

  UpdateOrderWithShippingInfoRequest({required this.orderId, required this.shippingAddress, required this.shippingFee, required this.shippingPhoneNumber});
  Map<String, dynamic> toJson() {
    return <String, dynamic>{"orderId": orderId,
  "shippingFee": shippingFee,
  "shippingAddress": shippingAddress,
  "shippingPhoneNumber": shippingPhoneNumber};
  }
}

File: FoodOrderApis/OrderService/Data/Requests/UpdateOrderWithShippingInfoRequest.cs
namespace OrderService.Data.Requests;

public class UpdateOrderWithShippingInfoRequest
{
    public int OrderId { get; set; }
    public decimal ShippingFee { get; set; }
    public string ShippingAddress { get; set; }
    public string ShippingPhoneNumber { get; set; }
}